listenbrainz/spark/troi_bot.py

```diff
@@ -0,0 +1,22 @@
+""" This function contains code to run the various troi-bot functions after
+    recommendations have been generated.
+"""
+from troi.core import generate_playlist
+import config
+
+USER_TO_PROCESS = ["mr_monkey", "rob", "akshaaatt", "Damselfish", "lucifer", "alastairp", "CatCat", "atj"]
+
+def run_post_recommendation_troi_bot(user):
+    """
+        Top level function called after spark CF recommendations have been completed.
+    """
+
+    if user in USERS_TO_PROCESS:
+        make_playlist_from_recommendations(user)
+        # Add others here
+
+def make_playlist_from_recommendations(user):
+
+    token = config.WHITELISTED_AUTH_TOKENS[0]
```

Comment:
We should use `flask.current_app` to get config here.

Reply:
Is this code running in a flask context??

Reply:
Yup, entire spark reader thus all handlers run in flask context (https://github.com/metabrainz/listenbrainz-server/blob/28b27f9a9bd3af933aa42d53848391ecebe4152f/listenbrainz/spark/spark_reader.py#L127)

Reply:
Updated.